src/graphql-schema/types.ts

```diff
@@ -1,4 +1,5 @@
 export interface Credentials {
   email: string;
   password: string;
+  rememberMe: boolean;
```

Comment:
No Typescript tbm tem indicativo de opcional com `?`:

```suggestion
export interface Credentials {
  email: string;
  password: string;
  rememberMe?: boolean;
```